src/features/session_trace/aggregate/index.js

```diff
@@ -111,22 +111,31 @@ export class Aggregate extends AggregateBase {
       this.abort(3)
       return
     }
+
+    // stage timings if PO isn't supported, this checks resourcetiming buffer every harvest.
+    if (!SUPPORTS_PERFORMANCE_OBSERVER) {
```

Comment:
We shouldn't add backwards version oriented code (browser versions that didn't support POs).

On another note, I don't believe there are existing tests that checks for duplicated resource across harvests which is relevant to this kind of change.

Reply:
`We shouldn't add backwards version oriented code (browser versions that didn't support POs).`
this is a direct port-over from the previous code.  We can remove if you think its no longer necessary